<launch>
        <arg name="ROBOT_NAME" default="turtlebot3"/>
        <!--megarover turtlebot3-->
        <arg name="IS_SIMULATOR" default="true"/>

    <node pkg="potbot" name="potbot_co" type="potbot_co" output="screen">
        <param name="ROBOT_NAME" value="$(arg ROBOT_NAME)"/>
        <param name="IS_SIMULATOR" value="$(arg IS_SIMULATOR)"/>
        <param name="PUBLISH_COMMAND" value="true"/>
        <param name="PATH_TRACKING_MARGIN" value="0.3"/>
    </node>
</launch>
